P5-251: Updated snippets to match code (#175)
aau-hcl-p5/main: report/5.Solution/Object_Detection.tex

\subsection{Object Localization}\label{solution:ObjectLocalization}
As mentioned in \autoref{flatcontrollerimplementation} there are multiple implemented options for object localization algorithms.
The object localization has to return the direction in which the target is, and whether target is covering the center of the screen.

> This is for the purpose of whether to turn on the laser.

Reviewer: ```suggestion
 This is used for determining whether the laser should be turned on or off.
```
